CHANGELOG.md

```diff
@@ -3,13 +3,17 @@
 All notable changes to this project will be documented in this file. This library adheres to a versioning policy described in [the README](./README.md#versioning). The public API of this library consists of the functions exported in [h3core.js](./lib/h3core.js).
 
 ## [Unreleased]
-- *None*
+### Fixed
+- Updated the core library to v3.1.1, including fixes for `polyfill` and `h3SetToMultiPolygon` (#20)
```

Comment:
#20 doesn't exist

Reply:
Oh, good catch, will update to #19. Not sure why I thought this PR was going to be 20.